main/inc/lib/hook/HookLearningPathEnd.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/* For licensing terms, see /license.txt */
```

Comment:
Tenía entendido que ahora era necesario poner una línea blanca entre `<?php` y `/* For licensing terms, ...`